docs/configuration.md

```diff
@@ -353,6 +355,18 @@ Overrides the number of automatically determined test shards for Flank to use. U
     testShards.set(5)
     ```
 
+### maxTestShards
+The maximum number of shards.
+
+=== "Groovy"
+``` groovy
```

Comment:
just a subtle thing here, can you add 4 space tabs here to match the rest of the styling? otherwise the markdown formatting on the documentation website gets screwed up.
```suggestion
    ``` groovy
    maxTestShards = 8
    ```
```